src/cosmoz-dropdown.ts

```diff
@@ -104,16 +128,17 @@ const Dropdown = (host: HTMLElement & DropdownProps) => {
 		${when(
 			active,
 			() =>
-				html` <cosmoz-dropdown-content
-					id="content"
-					part="content"
-					exportparts="wrap, content"
-					.anchor=${anchor}
-					.placement=${placement}
-					.render=${render}
-				>
-					<slot></slot>
-				</cosmoz-dropdown-content>`
+				html`<div ${ref(popoverChanged)} popover id="content">
```

Comment:
Can't we set the `popover`  attribute on the `cosmoz-dropdown-content` element and avoid creating another div  ?

Reply:
I tried just now, but when I click the anchor nothing shows up.

Reply:
I think I managed to get it right in commit 7dcc348